code/game/machinery/decontamination.dm

```diff
@@ -132,7 +132,7 @@
 			flick("tube_up", src)
 			decon.stop()
 			playsound(src, 'sound/machines/decon/decon-up.ogg', 100, TRUE)
-			sleep(8)
+			sleep(8) //why
```

Comment:
this is to wait for the "tube_up" to finish its animation before opening the door

Reply:
also can you increase it by 0.5

Reply:
we shouldn't be using sleep anymore

Reply:
give me something better